src/main/kotlin/com/sourcegraph/cody/listeners/CodySelectionInlayManager.kt

```diff
@@ -0,0 +1,162 @@
+package com.sourcegraph.cody.listeners
+
+import com.intellij.openapi.actionSystem.KeyboardShortcut
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.EditorCustomElementRenderer
+import com.intellij.openapi.editor.Inlay
+import com.intellij.openapi.editor.InlayModel
+import com.intellij.openapi.editor.colors.EditorColors
+import com.intellij.openapi.editor.event.SelectionEvent
+import com.intellij.openapi.editor.markup.TextAttributes
+import com.intellij.openapi.keymap.KeymapManager
+import com.intellij.openapi.util.Disposer
+import com.intellij.util.ui.UIUtil
+import java.awt.Font
+import java.awt.Graphics
+import java.awt.Graphics2D
+import java.awt.Rectangle
+import java.awt.event.KeyEvent
+import java.awt.geom.GeneralPath
+import java.util.*
+
+class CodySelectionInlayManager {
+
+  private var currentInlay: Inlay<*>? = null
+  private var inlayBounds: Rectangle? = null
```

Comment:
How is it used? 🤔

Reply:
Yep good call -- wasn't used; I removed it.